src/main/java/com/gmail/inverseconduit/commands/sets/CoreBotCommands.java

```diff
@@ -15,15 +15,16 @@
 
 import com.gmail.inverseconduit.AppContext;
 import com.gmail.inverseconduit.BotConfig;
-import com.gmail.inverseconduit.bot.Program;
 import com.gmail.inverseconduit.chat.ChatInterface;
+import com.gmail.inverseconduit.chat.ChatWorker;
 import com.gmail.inverseconduit.chat.Subscribable;
 import com.gmail.inverseconduit.commands.CommandHandle;
+import com.gmail.inverseconduit.commands.providers.canned_comments.CannedComments;
```

Comment:
this line prevents compilation of the pull-request currently. It slipped in when I didn't have this and https://github.com/Vogel612/JavaBot/tree/canned-comments separate yet